site/app/controllers/admin/SqlToolboxController.php

```diff
@@ -24,7 +24,15 @@ class SqlToolboxController extends AbstractController {
      * @Route("/courses/{_semester}/{_course}/sql_toolbox", methods={"GET"})
      */
     public function showToolbox(): WebResponse {
-        return new WebResponse(SqlToolboxView::class, 'showToolbox');
+        $organizedTables = [];
+        //Loop through and create a 2d array that holds all columns for each table name.
+        foreach ($this->core->getQueries()->getCourseSchemaTables() as $table) {
+            if (!isset($organizedTables[$table['table_name']])) {
+                $organizedTables[$table['table_name']] = [];
+            }
+            $organizedTables[$table['table_name']][] = $table['column_name'] . " - " . $table['data_type'];
```

Comment:
So, I think a better construct would be to do:
```php
            $organizedTables[$table['table_name']][] = [
                'name' => $table['column_name'],
                'type' => $table['data_type'],
            ];
```
and then in the twig template, we combine the columns to do `{{column.name}} - {{column.type}}`. The purpose behind this is in trying to make sure we do view stuff as close to the view layer as possible, and let the controllers and models interact with the data as close to its source as possible. This allows for easier augmentation of the data object with more information if needed here, as well as we could re-use the data model elsewhere more easily.